layouts/_default/index.json.json

```diff
@@ -0,0 +1,11 @@
+{{- $.Scratch.Add "index" slice -}}
```

Comment:
What is this file?

Reply:
This creates the `featured-posts.json` file.